packages/react/src/internal/FeatureFlags.js

```diff
@@ -9,9 +9,22 @@
  * This file contains the list of the default values of compile-time feature flags.
  */
 
+/**
+ * Helper function for rendering feature flag
+ *
+ * @param {string} flag identifies which flag is used
+ * @param {string} jsx pass in the jsx to render conditionally
+ * @private
+ * @returns {object} JSX object
+ */
+export const featureFlag = (flag, jsx) => {
```

Comment:
@photodow I dig this!

cc: @annawen1 @kennylam

Reply:
can we pull this out and into the `utilities` package instead? Would want to use this for my lead space component in patterns

Reply:
done and done